wrappers/nodejs/test/test-config.js

```diff
@@ -175,4 +209,105 @@ describe('Config test', function() {
       config.enableAllStreams(1);
     });
   });
+
+  it('Testing method enableDeviceFromFile - 0 argument', () => {
+    assert.throws(() => {
+      config.enableDeviceFromFile();
+    });
+  });
+
+  it('Testing method enableDeviceFromFile - 2 argument', () => {
+    assert.throws(() => {
+      config.enableDeviceFromFile(1, 1);
+    });
+  });
+
+  it('Testing method enableDeviceFromFile - argument with number', () => {
+    assert.throws(() => {
+      config.enableDeviceFromFile(1);
+    });
+  });
+
+  it('Testing method enableDeviceFromFile - with file', () => {
+    createRrdFile();
+    assert.doesNotThrow(() => {
+      config.enableDeviceFromFile(fileName);
+    });
+  });
+
+  it('Testing method enableDeviceFromFile - with invalid argument', () => {
+    assert.throws(() => {
+      config.enableDeviceFromFile('dummy');
+    });
+  });
+
+  it('Testing method enableDeviceFromFile - after enableRecordToFile', () => {
+    assert.doesNotThrow(() => {
+      config.enableRecordToFile(fileName);
+      pipeline.start(config);
+    });
+    assert.throws(() => {
+      config.enableDeviceFromFile(fileName);
+    });
+  });
+
+  it('Testing method enableRecordToFile - 0 argument', () => {
+    assert.throws(() => {
+      config.enableRecordToFile();
+    });
+  });
+
+  it('Testing method enableRecordToFile - two arguments', () => {
+    assert.throws(() => {
+      config.enableRecordToFile(fileName, fileName);
+    });
+  });
+
+  it('Testing method enableRecordToFile - argument with number', () => {
+    assert.throws(() => {
+      config.enableRecordToFile(1);
+    });
+  });
+
+  it('Testing method enableRecordToFile - without pipeline.start', () => {
+    assert.doesNotThrow(() => {
+      config.enableRecordToFile(fileName);
+    });
+    assert.equal(fs.existsSync(fileName), false);
+  });
+
+  it('Testing method enableRecordToFile - value result', () => {
+    assert.doesNotThrow(() => {
+      config.enableRecordToFile(fileName);
+      pipeline.start(config);
+    });
+    assert.equal(fs.existsSync(fileName), true);
+    assert.doesNotThrow(() => {
+      pipeline.waitForFrames();
+    });
+  });
+
+  it('Testing method enableRecordToFile - then enableDeviceFromFile', () => {
+    assert.doesNotThrow(() => {
+      config.enableRecordToFile(fileName);
+      pipeline.start(config);
+      pipeline.waitForFrames();
```

Comment:
Please add check point to prove waitForFrame works well.
e.g. returned frameSet.depthFrame instanceof rs2.VideoFrame